src/main/fc/fc_rc.c

```diff
@@ -239,6 +237,105 @@ FAST_CODE FAST_CODE_NOINLINE void processRcCommand(void)
         rcInterpolationStepCount = 0; // reset factor in case of level modes flip flopping
     }
 
+    DEBUG_SET(DEBUG_RC_INTERPOLATION, 2, rcInterpolationStepCount);
+
+    return updatedChannel;
+
+}
+
+#ifdef USE_RC_SMOOTHING_FILTER
+FAST_CODE FAST_CODE_NOINLINE uint8_t processRcSmoothingFilter(void)
```

Comment:
Why is this `FAST_CODE_NOINLINE`? This will just make us slower and cost us more ITCM_RAM (worst case).

Reply:
Looks like the attributes were copy-pasted.